codec/src/main/java/io/netty/handler/codec/HeadersUtils.java

```diff
@@ -231,7 +231,9 @@ public boolean contains(Object o) {
         private void fillArray(Object[] arr) {
             Iterator<T> itr = allNames.iterator();
             for (int i = 0; i < size(); ++i) {
```

Comment:
i think the iterator says you need to call hasNext()?

Reply:
@carl-mastrangelo yep this would be more correct... let me do this.